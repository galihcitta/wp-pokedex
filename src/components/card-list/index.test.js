import React from "react";
import CardList from "./index";

describe("Card List Component", () => {
    it("renders and create snapshot <CardList />", () => {
        expect(
            toJson(
                shallow(<CardList radius="circle" size="lg" name="BELARUS" />),
            ),
        ).toMatchSnapshot();
    });
});
