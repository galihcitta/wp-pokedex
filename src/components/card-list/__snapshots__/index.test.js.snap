// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card List Component renders and create snapshot <CardList /> 1`] = `
<styled.div>
  <styled.div>
    <ThumbAvatar
      image="dummy-square.jpg"
      radius="circle"
      size="lg"
    />
  </styled.div>
  <styled.div>
    <styled.p
      style={
        Object {
          "textTransform": "capitalize",
        }
      }
    >
      BELARUS
    </styled.p>
    <IosArrowForward
      beat={false}
      color="#424242"
      fontSize="24px"
      rotate={false}
      shake={false}
      style={Object {}}
    />
  </styled.div>
</styled.div>
`;
